tests/test_antiweb.py

```diff
@@ -30,9 +30,10 @@ class Test_Antiweb(unittest.TestCase):
     #This function will be called before each Test. It copies the file to process into the temp directory of the test
     #@code
     def setUp(self):
-        self.directory = "docs"
+        self.tests_dir = os.path.dirname(os.path.abspath(__file__))
+        self.doc_dir = "docs"
         self.test_dir = tempfile.mkdtemp()
-        self.origin_path = os.path.join(os.getcwd(), os.path.join("unittest", "small_testfile.py"))
+        self.origin_path = os.path.join(self.tests_dir, os.path.join("unittest", "small_testfile.py"))
```

Comment:
* The 'unittest' and 'unittest_rst' folders are not required.
* They help to partition the 'tests' directory into more manageable pieces of data input/output for error tracing (both 'unittest' and 'unittest_rst' contain an index.rst file which would overwrite one another).
* If you take a look into 'tests', it already contains a lot of input files. I personally would put all those files into 'tests/data/...' and create subdirectories for better structuring.